### ISSUE TITLE
Keep EKS and Kubernetes up to date

### ISSUE THREAD

**Issue by adborden** [GSA/datagov-ckan-multi#143]
## User Story
As an operator, I want Kubernetes running at a stable, supported version so that I am running an EKS supported version and my cluster has the latest security patches.

We are currently running Kubernetes 1.10 which is no longer supported by EKS.
> Please update any 1.10 clusters to version 1.11 or higher in order to avoid service interruption.
> Learn more: https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html

## Acceptance Criteria
- [ ] Kubernetes is automatically updated

**Comment by aluminiumgeek** [comment]
It is safe to upgrade cluster version. There should be a button in AWS console that will initiate the upgrade and will roll the new Kubernetes version on all nodes without (or with minimal) downtime.

**Comment by jqnatividad** [comment]
Hi @adborden, can you confirm this meets the acceptance criteria?

**Comment by adborden** [comment]
No, it does not meet the acceptance criteria. This story is about making the updates automated, or at least generating a notification so we  know a manual action is required. Currently we have no way of knowing that EKS is outdated.

In production, I would like operators to have limited or no write access, so using the AWS console wouldn't be an option.

I've upgraded EKS to 1.12 and continue to manually update this environment for now.

**Comment by adborden** [comment]
Let's kick this back to backlog until we are ready to pick this up.

**Comment by kimwdavidson** [comment]
Needs a bit more research to be estimated. Moving to needs spec

**Comment by pwalsh** [comment]
@adborden 

Hey. Different cloud providers deal with this differently from what I know. As we use managed k8s (on GCP for example), some updates are done automatically by the provider, but then there are other cases that require cluster updates, for example, linux vulnerabilities, or types of updates that require manual action and so on. I am not sure that such updates should be automatic - they probably need to be tested in a sandbox first, or have *some* type of human verification? Basically, I get the need but I think we can sharpen the user story to understand the exact conditions in which we expect automated updates, and then we can explore of/how this is possible feasible on AWS, GCP, etc.
